### PR TITLE
Works

### PR DESCRIPTION
**Pull request by lu-jim**
## Second PR for this project
- Add Works section using CSS Grid
- Add About section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,6 +50,140 @@ <h2 class="connect"> <a href="#">Let's connect</a></h2>
...
+            <a href="#" class="sec-category">2015</a>
+          </div>
+        </div>
+
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
```
Comment (Alexoid1) [inline index.html:99]: - [ ] Kindly replace `div` with `ul` tag and list the technologies with `li` tags.

File: index.html [hunk 2]
```diff
@@ -49,6 +50,140 @@ <h2 class="connect"> <a href="#">Let's connect</a></h2>
...
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">2015</a>
+          </div>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
```
Comment (Alexoid1) [inline index.html:120]: - [ ] Kindly replace `div` with `ul` tag and list the technologies with `li` tags.

File: index.html [hunk 3]
```diff
@@ -49,6 +50,140 @@ <h2 class="connect"> <a href="#">Let's connect</a></h2>
...
```
Comment (Alexoid1) [inline index.html:142]: - [ ] Kindly replace `div` with `ul` tag and list the technologies with `li` tags.

File: index.html [hunk 4]
```diff
@@ -49,6 +50,140 @@ <h2 class="connect"> <a href="#">Let's connect</a></h2>
...
```
Comment (Alexoid1) [inline index.html:75]: - [ ] Kindly replace div with ul tag and list the technologies with li tags.

File: index.html [hunk 5]
```diff
@@ -49,6 +50,140 @@ <h2 class="connect"> <a href="#">Let's connect</a></h2>
           </ul>
         </div>
       </section>
+    </div>
+
+
+    <section class="works">
+      <article class="card card-1">
```
Comment (Alexoid1) [inline index.html:57]: - [ ] Kindly make the image and paragraph responsive for mobile devices.
Captura de pantalla_2021-09-22_17-19-38

File: index.html [hunk 6]
```diff
@@ -49,6 +50,140 @@ <h2 class="connect"> <a href="#">Let's connect</a></h2>
...
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
+
+      <article class="card card-2">
```
Comment (Alexoid1) [inline index.html:79]: - [ ] Kindly make the image and paragraph responsive for mobile devices.
Captura de pantalla_2021-09-22_17-19-56

File: index.html [hunk 7]
```diff
@@ -49,6 +50,140 @@ <h2 class="connect"> <a href="#">Let's connect</a></h2>
...
+
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
+      <article class="card card-3">
```
Comment (Alexoid1) [inline index.html:102]: - [ ] Kindly make the image and paragraph responsive for mobile devices.
Captura de pantalla_2021-09-22_17-20-16

File: index.html [hunk 8]
```diff
@@ -49,6 +50,140 @@ <h2 class="connect"> <a href="#">Let's connect</a></h2>
...
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
+
+      <article class="card card-4">
```
Comment (Alexoid1) [inline index.html:124]: - [ ] Kindly make the image and paragraph responsive for mobile devices.

File: index.html [hunk 9]
```diff
@@ -49,6 +50,140 @@ <h2 class="connect"> <a href="#">Let's connect</a></h2>
...
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
+    </section>
+    <section class="about-section">
+      <h2 class="about-title title">
```
Comment (Alexoid1) [inline index.html:147]: - [ ] Kindly fix the styles in the about section, the paragraph is not responsive for mobile devices.
- [ ] Kindly add margin to the section container.
- [ ] Make the containers of every technology responsive for mobile devices.
Captura de pantalla_2021-09-22_17-48-40

File: index.html [hunk 10]
```diff
@@ -23,12 +25,11 @@
...
-      <h1 class="title">
-         I'm Luis <br>Glad to see you!
-      </h1>
+    <div class="headline-background">
+      <section class="headline">
+        <h1 class="title">
+          I'm Luis <br>Glad to see you!
+        </h1>
         <div class="header-content">
           <p class="descr">I’m a software developer! I can help you build a product, feature or website. Look through
```
Comment (Alexoid1) [inline index.html:34]: -[OPTIONAL] Kindly add some margin to this section and make the paragraph responsive for mobile devices.
Captura de pantalla_2021-09-22_17-56-19

**Review by Alexoid1 — CHANGES_REQUESTED** [review]
# Hi @lu-jim,

Impressive project!
You are doing a great job so far, and the project is looking good.
- No linter errors
- Follow Github-flow guidelines.
- Descriptive README.md file.
- Impressive project is almost the exact same design as the Figma template.

## Required Changes :recycle:
 - For the project to get approved, also implement the changes mentioned below.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by carlos-ssh — APPROVED** [review]
hi @lu-jim , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**